pkg/model/image_manager.go

```diff
@@ -0,0 +1,66 @@
+package model
+
+import (
+	"os"
+	"runtime"
+)
+
+const (
+	KeycloakImage         = "IMAGE_KEYCLOAK"
+	RHSSOImageOpenJ9      = "IMAGE_RHSSO_OPENJ9"
+	RHSSOImageOpenJDK     = "IMAGE_RHSSO_OPENJDK"
+	RHSSOImage            = "IMAGE_RHSSO"
+	KeycloakInitContainer = "IMAGE_KEYCLOAK_INIT_CONTAINER"
+	RHMIBackupContainer   = "IMAGE_RHMI_BACKUP_CONTAINER"
+	PostgresqlImage       = "IMAGE_POSTGRESQL"
+
+	DefaultKeycloakImage         = "quay.io/keycloak/keycloak:9.0.2"
+	DefaultRHSSOImageOpenJ9      = "registry.access.redhat.com/redhat-sso-7/sso73-openshift:1.0-15"
```

Comment:
Is this _default_ value necessary? Either there is a cost associated with maintaining this default, or the operator would pick this aging image.
Could it be preferably removed for RHSSO and set the image mandatorily from outside in the manifest / on startup rather than hardcoding it?
cc: @stianst

Reply:
In the longer term - yes. But I'm afraid we can't do it just yet. 

The RHMI Team uses this Operator with `RHSSO` profile. If we removed this now, we will put them in a very uncomfortable situation and force them to switch to a new RHSSO Operator immediately. If there's any bug in the initial release of the RHSSO Operator, we would all be in trouble. 

So, in the long term we should remove it. In short and mid term - we should keep it.